### PR TITLE
#83 add event view web

### PR DESCRIPTION
**Pull request by christianrm02**

### CODE REVIEW

File: src/views/CreateEvent.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
+<template>
+  <v-col>
+    <h1 style="color: #ff6961" class="my-5 ml-5">Creació d'un esdeveniment</h1>
+    <v-container class="d-flex justify-center align-center">
+      <v-col cols="12">
+        <!-- ========================== AVATAR Y SETTINGS ========================== -->
+        <v-card>
+
+    <v-form @submit="submitForm">
+      <p v-if="v$.formData.nom.$error" style="color: red;" class="mt-4 mb-4 ml-4 mr-4">Name can't be empty.</p>
```
Comment (Ericriera) [inline src/views/CreateEvent.vue:10]: en vez de usar todas estas clases, usa ma-4 tiene el mismo efecto

File: src/views/CreateEvent.vue [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      <v-text-field label="Nom" v-model="formData.nom" variant="outlined" class="mt-4 mb-4 ml-4 mr-4"></v-text-field>
+      <p v-if="v$.formData.descripcio.$error" style="color: red;" class="mt-4 mb-4 ml-4 mr-4">Descripció max characters=560.</p>
+      <v-textarea label="Descripció" v-model="formData.descripcio" variant="outlined" auto-grow class="mt-4 mb-4 ml-4 mr-4"></v-textarea>
+      <!-- <v-file-input
+        label="Selecciona imatge (jpg o png)"
+        :accept="'image/jpeg, image/png'"
+        variant="outlined"
+        v-model="formData.imatge"
+      ></v-file-input> -->
```
Comment (Ericriera) [inline src/views/CreateEvent.vue:19]: no dejes comentado código

**Review by Ericriera — CHANGES_REQUESTED** [review]
Sale este error al hacer el deploy: 10:31:30 AM: [vite]: Rollup failed to resolve import "axios" from "/opt/build/repo/src/views/CreateEvent.vue".
10:31:30 AM: This is most likely unintended because it can break your application at runtime.
10:31:30 AM: If you do want to externalize this module explicitly add it to
10:31:30 AM: `build.rollupOptions.external`
10:31:30 AM: error during build:
10:31:30 AM: Error: [vite]: Rollup failed to resolve import "axios" from "/opt/build/repo/src/views/CreateEvent.vue".
10:31:30 AM: This is most likely unintended because it can break your application at runtime.
10:31:30 AM: If you do want to externalize this module explicitly add it to
10:31:30 AM: `build.rollupOptions.external`
10:31:30 AM:     at viteWarn (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-2b82a1ce.js:48205:27)
10:31:30 AM:     at onRollupWarning (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-2b82a1ce.js:48237:9)
10:31:30 AM:     at onwarn (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-2b82a1ce.js:47965:13)
10:31:30 AM:     at file:///opt/build/repo/node_modules/rollup/dist/es/shared/node-entry.js:24276:13
10:31:30 AM:     at Object.logger [as onLog] (file:///opt/build/repo/node_modules/rollup/dist/es/shared/node-entry.js:25950:9)
10:31:30 AM:     at ModuleLoader.handleInvalidResolvedId (file:///opt/build/repo/node_modules/rollup/dist/es/shared/node-entry.js:24862:26)
10:31:30 AM:     at file:///opt/build/repo/node_modules/rollup/dist/es/shared/node-entry.js:24822:26
10:31:30 AM: ​
10:31:30 AM: "build.command" failed 

Intenta arreglerlo y si necessitas más ayuda me dices y nos lo miramos

File: src/views/CreateEvent.vue (hunk 2 above)
Comment (christianrm02) [reply src/views/CreateEvent.vue]: esto lo dejo porq el tema de las imagenes se tendrán q tratar, pero el back no está preparado

**Review by Ericriera — APPROVED** [review]
Ready to merge

**Pull request merged**